core/templates/dev/head/components/state-editor/state-interaction-editor/state-interaction-editor.directive.spec.ts

```diff
@@ -42,6 +42,13 @@ describe('State Interaction controller', function() {
   describe('StateInteraction', function() {
     beforeEach(function() {
       angular.mock.module('oppia');
+      angular.mock.module(function($provide) {
+        $provide.value('ExplorationFeaturesService', {
+          isPlaythroughRecordingEnabled: function() {
+            return false;
```

Comment:
Can you document why you set this to `false`?

Reply:
Added a comment. PTAL!